### PR TITLE
Improve Snappy image printing

### LINKED ISSUE #1419: Snappy image deformed
### Problem
When preparing official docker image for Snap, I have encountered an issue concerning printing Snappy image along with snapteld logs. That is when when TTY is not attached this is what happens to Snappy:

```
time="2016-12-06T13:14:09Z" level=info msg="module started" _module=snapteld block=main snap-module=REST 
time="2016-12-06T13:14:09Z" level=info msg="setting plugin trust level to: disabled" 
time="2016-12-06T13:14:09Z" level=info msg="snapteld started\n                                        ss                  ss\n\t                             odyssyhhyo         oyhysshhoyo\n               
                  ddddyssyyysssssyyyyyyyssssyyysssyhy+-\n                           ssssssooosyhhysssyyyyyyyyyyyyyyyyyyyyyyyyyyyyssyhh+.\n                          ssss lhyssssssyyyyyyyyyyyyyyyyyyyyyyyyy$
yyyyyyyyyssydo\n                sssssssssshhhhs lsyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyshh.\n           ssyyyysssssssssssyhdo syyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyd.\n       syyyyyyyyhhy$
yyyyyyyyyyhdo syyyyyyyyyyyyydddhyyyyyyyyyyyyyhhhyyyyyyyyyyyyhh\n     ssyyyyyyyh  hhyyyyyyyyyyyyhdo syyyyyyyyyyyddyddddhhhhhhhhdddhhddyyyyyyyyyyyydo\n     ddyyyyyyh |  hyyyyyyyyyyyyydds syyyyyyyhhdhyyyhhh$
hhhhhhyyyyyyhhdhyyyyyyyyyyym+\n     dhyyyyyyyhhhhyyyyyyyyyyyyyyhmds shyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyydyyl\n     dhddhyyyyyyhdhyyyyyyyyyyyyyydhmo yhyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyy$
dhsh\n     dhyyhysshhdmdhyyyyyyyyyyyyyyhhdh  hhyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyhhdyoohmh ylo\n      yy       dmyyyyyyyyyyyyyyyyhhhm  odhyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyhdhsoshdyyhdddy ylo\n    
        odhhyyyyyyyyyyyyyyyyyyhhdy  oyhhhyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyhhmhyhdhhyyyyyhddoyddy ylo\n           dddhhyyyyyyyyyyyyyyyhhhyhhdhs ooosydhhhhyyyyyyyyyyyyyyyyyyyyhhhhhyso+oymyyyyyyyyyyyhhddydmmy$
s\n             ohdyyyyyyyyyyyyyyyhdhyyhhhddyoooohhhhhhhhhhhhhhhhhhhhhdhhyysooosyhhhhhyhhhhhhyyyhyyyhhhhddyy\n                dyhyyyyyyyyyyyyydhyyyyyhhdddoooooooooooooooooooooooyysyyhddddhhyyyyydmddddddd$
mddddhyyy\n               dmmmmoNddddddmddhhhhyyyyyhhhdddddddhhhhhhhhhdddddddddhhhyyyyyyyyyyhNmmmooooooooyyy\n                     Nhhhhhhhdmmddmhyyyyyyyyyhhhhhhhhhhhhhhhhhhhhyyyyyyyyyyyyyyyhhm\n         
            NhhhhhhhhhdmyhdyyyyyyyyyyyyhyyyyyyyyyyyyyyyhhhdhyyyyyyyyyyyhhN\n                     NhyyyyyyyyyN dmyyyyyyyyyyhdmdhhhhhhhhhhdhhmmmmN NyyyyyyyyyyhhN\n                     NhyyyyyyyyyN  Nyyyyyy$
yyhhm               NmddmH Nyyyyyyyyyhdm\n       .d8888b.      dmomomommmmh  dhhhhhhyyhhmh               NddddmH Nyyyyyyyyhdh\n      d88P  Y88b                   dmomomommmmmh                dmomoH dmomo$
ommmmh\n      Y88b.\n      \"Y888b.   88888b.   8888b.  88888b.\n         \"Y88b. 888 \"88b     \"88b 888 \"88b\n           \"888 888  888 .d888888 888  888\n     Y88b  d88P 888  888 888  888 888 d88P\n  
    \"Y8888P\"  888  888 \"Y888888 88888P\"\n
```
This will be always visible in docker logs when running Snap in a container i.e. like this:
`docker run -d snap ` 
without tty attaching flag (`docker run -t -d snap`). 

### Significance
That is rather important, as it will be the first thing printed in the console when user runs Snap docker image and it will be visible in logs. And `docker logs -f snap` is the first place user goes to check whether Snap container has started correctly.

### Cause
The cause of this issue is that we use https://github.com/sirupsen/logrus to print the turtle (https://github.com/intelsdi-x/snap/blob/6b7404dcfcb6999165cb1ca56047f4c346c067b0/snapteld.go#L471) and as it is stated in logrus documentation the format of the logs changes (see https://github.com/sirupsen/logrus/blob/master/README.md):

> With the default log.SetFormatter(&log.TextFormatter{}) when a TTY is not attached, the output is compatible with the logfmt format.

### Possible solutions
1. Use regular fmt.Print() instead of logrus to print Snappy image. I verified, it helps - after this change logs without tty attached look like this:
```
time="2016-12-06T15:25:38Z" level=info msg="REST started" _block=start _module="_mgmt-rest" 
time="2016-12-06T15:25:38Z" level=info msg="module started" _module=snapteld block=main snap-module=REST 
time="2016-12-06T15:25:38Z" level=info msg="setting plugin trust level to: disabled" 
time="2016-12-06T15:25:38Z" level=info msg="snapteld started" _module=snapteld block=main 

                                        ss                  ss
                                     odyssyhhyo         oyhysshhoyo
                                 ddddyssyyysssssyyyyyyyssssyyysssyhy+-
                           ssssssooosyhhysssyyyyyyyyyyyyyyyyyyyyyyyyyyyyssyhh+.
                          ssss lhyssssssyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyssydo
                sssssssssshhhhs lsyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyshh.
           ssyyyysssssssssssyhdo syyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyd.
       syyyyyyyyhhyyyyyyyyyyyyhdo syyyyyyyyyyyyydddhyyyyyyyyyyyyyhhhyyyyyyyyyyyyhh
     ssyyyyyyyh  hhyyyyyyyyyyyyhdo syyyyyyyyyyyddyddddhhhhhhhhdddhhddyyyyyyyyyyyydo
     ddyyyyyyh |  hyyyyyyyyyyyyydds syyyyyyyhhdhyyyhhhhhhhhhhyyyyyyhhdhyyyyyyyyyyym+
     dhyyyyyyyhhhhyyyyyyyyyyyyyyhmds shyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyydyyl
     dhddhyyyyyyhdhyyyyyyyyyyyyyydhmo yhyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyhdhsh
     dhyyhysshhdmdhyyyyyyyyyyyyyyhhdh  hhyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyhhdyoohmh ylo
      yy       dmyyyyyyyyyyyyyyyyhhhm  odhyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyhdhsoshdyyhdddy ylo
            odhhyyyyyyyyyyyyyyyyyyhhdy  oyhhhyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyhhmhyhdhhyyyyyhddoyddy ylo
           dddhhyyyyyyyyyyyyyyyhhhyhhdhs ooosydhhhhyyyyyyyyyyyyyyyyyyyyhhhhhyso+oymyyyyyyyyyyyhhddydmmyys
             ohdyyyyyyyyyyyyyyyhdhyyhhhddyoooohhhhhhhhhhhhhhhhhhhhhdhhyysooosyhhhhhyhhhhhhyyyhyyyhhhhddyy
                dyhyyyyyyyyyyyyydhyyyyyhhdddoooooooooooooooooooooooyysyyhddddhhyyyyydmdddddddmmddddhyyy
               dmmmmoNddddddmddhhhhyyyyyhhhdddddddhhhhhhhhhdddddddddhhhyyyyyyyyyyhNmmmooooooooyyy
                     Nhhhhhhhdmmddmhyyyyyyyyyhhhhhhhhhhhhhhhhhhhhyyyyyyyyyyyyyyyhhm
                     NhhhhhhhhhdmyhdyyyyyyyyyyyyhyyyyyyyyyyyyyyyhhhdhyyyyyyyyyyyhhN
                     NhyyyyyyyyyN dmyyyyyyyyyyhdmdhhhhhhhhhhdhhmmmmN NyyyyyyyyyyhhN
                     NhyyyyyyyyyN  Nyyyyyyyyyhhm               NmddmH Nyyyyyyyyyhdm
       .d8888b.      dmomomommmmh  dhhhhhhyyhhmh               NddddmH Nyyyyyyyyhdh
      d88P  Y88b                   dmomomommmmmh                dmomoH dmomomommmmh
      Y88b.
      "Y888b.   88888b.   8888b.  88888b.
         "Y88b. 888 "88b     "88b 888 "88b
           "888 888  888 .d888888 888  888
     Y88b  d88P 888  888 888  888 888 d88P
      "Y8888P"  888  888 "Y888888 88888P"
                                  888
                                  888
                                  888  
```

2. Check if tty is attached and in case it's not - don't print Snappy. https://rosettacode.org/wiki/Check_output_device_is_a_terminal#Go

### PR DESCRIPTION
**Pull request by mkuculyma**
Fixes #1419

Summary of changes:
- Changed the way of printing Snappy with snapteld logs - logrus function changed to fmt.Println()

Testing done:
-  small tests

@intelsdi-x/snap-maintainers

### CODE REVIEW

File: snapteld.go [hunk 1]
```diff
@@ -467,12 +467,12 @@ func action(ctx *cli.Context) error {
...
 		}
 	}
-
 	log.WithFields(
 		log.Fields{
 			"block":   "main",
 			"_module": logModule,
-		}).Info("snapteld started", `
+		}).Info("snapteld started")
+	fmt.Println(`
```
Comment (IRCody) [inline snapteld.go]: We still need to respect the log level.

**Comment by kindermoumoute** [comment]
Sometime this happens:
```
                                        ss                  ss
	                             odyssyhhyo         oyhysshhoyo
                                 ddddyssyyysssssyyyyyyyssssyyysssyhy+-
                           ssssssooosyhhysssyyyyyyyyyyyyyyyyyyyyyyyyyyyyssyhh+.
                          ssss lhyssssssyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyssydo
                sssssssssshhhhs lsyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyshh.
           ssyyyysssssssssssyhdo syyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyd.
       syyyyyyyyhhyyyyyyyyyyyyhdo syyyyyyyyyyyyydddhyyyyyyyyyyyyyhhhyyyyyyyyyyyyhh
     ssyyyyyyyh  hhyyyyyyyyyyyyhdo syyyyyyyyyyyddyddddhhhhhhhhdddhhddyyyyyyyyyyyydo
     ddyyyyyyh |  hyyyyyyyyyyyyydds syyyyyyyhhdhyyyhhhhhhhhhhyyyyyyhhdhyyyyyyyyyyym+
time="2016-12-09T21:12:40Z" level=info msg="module started" _module=snapteld block=main snap-module=control
time="2016-12-09T21:12:40Z" level=info msg="scheduler started" _block=start-scheduler _module=scheduler
time="2016-12-09T21:12:40Z" level=info msg="auto discover path is enabled" _block=start-scheduler _module=scheduler
time="2016-12-09T21:12:40Z" level=info msg="autoloading tasks from: /opt/snap/plugins" _block=start-scheduler _module=scheduler
time="2016-12-09T21:12:40Z" level=info msg="module started" _module=snapteld block=main snap-module=scheduler
time="2016-12-09T21:12:40Z" level=info msg="Starting REST API on :8181" _module="_mgmt-rest"
time="2016-12-09T21:12:40Z" level=info msg="REST started" _block=start _module="_mgmt-rest"
time="2016-12-09T21:12:40Z" level=info msg="module started" _module=snapteld block=main snap-module=REST
time="2016-12-09T21:12:40Z" level=info msg="setting plugin trust level to: disabled"
time="2016-12-09T21:12:40Z" level=info msg="snapteld started" _module=snapteld block=main
     dhyyyyyyyhhhhyyyyyyyyyyyyyyhmds shyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyydyyl
     dhddhyyyyyyhdhyyyyyyyyyyyyyydhmo yhyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyhdhsh
     dhyyhysshhdmdhyyyyyyyyyyyyyyhhdh  hhyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyhhdyoohmh ylo
      yy       dmyyyyyyyyyyyyyyyyhhhm  odhyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyhdhsoshdyyhdddy ylo
            odhhyyyyyyyyyyyyyyyyyyhhdy  oyhhhyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyhhmhyhdhhyyyyyhddoyddy ylo
           dddhhyyyyyyyyyyyyyyyhhhyhhdhs ooosydhhhhyyyyyyyyyyyyyyyyyyyyhhhhhyso+oymyyyyyyyyyyyhhddydmmyys
             ohdyyyyyyyyyyyyyyyhdhyyhhhddyoooohhhhhhhhhhhhhhhhhhhhhdhhyysooosyhhhhhyhhhhhhyyyhyyyhhhhddyy
                dyhyyyyyyyyyyyyydhyyyyyhhdddoooooooooooooooooooooooyysyyhddddhhyyyyydmdddddddmmddddhyyy
               dmmmmoNddddddmddhhhhyyyyyhhhdddddddhhhhhhhhhdddddddddhhhyyyyyyyyyyhNmmmooooooooyyy
                     Nhhhhhhhdmmddmhyyyyyyyyyhhhhhhhhhhhhhhhhhhhhyyyyyyyyyyyyyyyhhm
                     NhhhhhhhhhdmyhdyyyyyyyyyyyyhyyyyyyyyyyyyyyyhhhdhyyyyyyyyyyyhhN
                     NhyyyyyyyyyN dmyyyyyyyyyyhdmdhhhhhhhhhhdhhmmmmN NyyyyyyyyyyhhN
                     NhyyyyyyyyyN  Nyyyyyyyyyhhm               NmddmH Nyyyyyyyyyhdm
       .d8888b.      dmomomommmmh  dhhhhhhyyhhmh               NddddmH Nyyyyyyyyhdh
      d88P  Y88b                   dmomomommmmmh                dmomoH dmomomommmmh
      Y88b.
      "Y888b.   88888b.   8888b.  88888b.
         "Y88b. 888 "88b     "88b 888 "88b
           "888 888  888 .d888888 888  888
     Y88b  d88P 888  888 888  888 888 d88P
      "Y8888P"  888  888 "Y888888 88888P"
                                  888
                                  888
                                  888
```

**Comment by jcooklin** [comment]
It's not clear to me what problem is being addressed here.

**Comment by mjbrender** [comment]
@jcooklin while the approach might not solve it as is, #1419 defines the problem in detail.

**Comment by jcooklin** [comment]
@mjbrender thanks, I don't know how I missed the fixes link.

**Comment by mkuculyma** [comment]
I'm closing this PR - Cody is right that this is not the proper way to fix #1419.

**Pull request closed without merging**